dvc/command/experiments.py

```diff
@@ -321,17 +328,30 @@ def experiments_table(
     return td
 
 
-def prepare_exp_id(kwargs):
+def prepare_exp_id(kwargs) -> "Text":
+    from rich.text import Text
+
     exp_name = kwargs["Experiment"]
-    ident = kwargs.get("ident_guide")
-    queued = kwargs.get("queued")
+    rev = kwargs["rev"]
+    typ = kwargs.get("typ", "baseline")
+
+    if typ == "baseline" or not exp_name:
+        text = Text(exp_name or rev)
+    else:
+        text = Text.assemble(rev, " [", (exp_name, "bold"), "]")
```

Comment:
Text is equivalent to `str` + styling, so it will be compatible with other tables too.